front/public/locales/de/operationalStudies/allowances.json

```
@@ -0,0 +1,28 @@
+{
+  "addAllowance": "Zuweisung hinzufügen",
+  "addLine": {
+    "beginPosition": "Von",
+    "distribution": "Verteilung",
+    "endPosition": "Nach",
+    "length": "Länge",
+    "value": "Wert"
+  },
+  "allowancesCount_one": "1 Zuweisung",
+  "allowancesCount_other": "{{ count }} Zuweisungen",
+  "allowancesCount_zero": "Keine Zuweisungen",
+  "cancel": "Abbrechen",
+  "deleteAllowance": "Zuweisung löschen",
+  "distribution": {
+    "linear": "linear",
+    "LINEAR": "Linear",
+    "mareco": "mareco",
+    "MARECO": "Mareco"
```

```suggestion
    "linear": "Linear",
    "LINEAR": "Linear",
    "mareco": "Mareco",
    "MARECO": "Mareco"
```

I'm not sure why we do have different keys, but they apparently don't mean to have different values... see [english](https://github.com/OpenRailAssociation/osrd/blob/dev/front/public/locales/en/operationalStudies/allowances.json) translations or [french](https://github.com/OpenRailAssociation/osrd/blob/dev/front/public/locales/fr/operationalStudies/allowances.json) ones